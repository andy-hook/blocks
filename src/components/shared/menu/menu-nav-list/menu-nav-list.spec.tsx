import React from "react"

import MenuNavList from "./menu-nav-list"
import { renderWithTheme } from "@test-utils"

describe("<MenuNavList />", () => {
  fit("renders correctly", () => {
    const tree = renderWithTheme(
      "light",
      <MenuNavList onClick={jest.fn} open={true} />
    )
    expect(tree).toMatchSnapshot()
  })
})
