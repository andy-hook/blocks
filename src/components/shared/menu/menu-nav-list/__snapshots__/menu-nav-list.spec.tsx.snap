// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuNavList /> renders correctly 1`] = `
<ul
  className="sc-bdVaJa jGpbbu"
>
  <li
    className="sc-bwzfXH gdYRbz"
  >
    <a
      className="sc-htpNat jzrKZq"
      href="/"
      onClick={[Function]}
    >
      Ten Blocks
    </a>
  </li>
  <li
    className="sc-bwzfXH gdYRbz"
  >
    <a
      className="sc-htpNat jzrKZq"
      href="/about/"
      onClick={[Function]}
    >
      About
    </a>
  </li>
</ul>
`;
